### PR TITLE
Add required prop to datepicker + submit type for buttons

### PR DESCRIPTION
**Pull request by tomdillien91**
## PR Checklist

This PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] A changelog entry has been added to CHANGELOG.md if necessary

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] react application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/form/src/Checkbox/Checkbox.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ type Props = {
   value?: string,
   description?: string,
   required?: boolean,
+	disabled?: boolean,
```
Comment (TriangleJuice) [inline packages/form/src/Checkbox/Checkbox.js]: Check hier alleen even de indenting nog.

Comment (tomdillien91) [reply packages/form/src/Checkbox/Checkbox.js]: @TriangleJuice Die tab indent sta lokaal juist bij mij. Ik heb de indruk dat de tab indents bij de pull requests hier soms helemaal fout staan.

File: packages/calendar/src/Datepicker/Nav.js [hunk 2]
```diff
@@ -27,17 +27,20 @@ class Nav extends Component {
 		return (
 			<React.Fragment>
 				<Button
+					type="button"
```
Comment (frankdsm) [inline packages/calendar/src/Datepicker/Nav.js]: htmlType="button"

File: packages/calendar/src/Datepicker/Nav.js [hunk 3]
```diff
@@ -27,17 +27,20 @@ class Nav extends Component {
...
 					onClick={onClickPrevious}
 					className={"a-button has-icon"}
 					icon={"angle-left"}/>
 
 				<Button
+					type="button"
```
Comment (frankdsm) [inline packages/calendar/src/Datepicker/Nav.js]: htmlType="button"

File: packages/calendar/src/Datepicker/Nav.js [hunk 4]
```diff
@@ -27,17 +27,20 @@ class Nav extends Component {
...
 					onClick={onClickViewType}
 					className={"m-datepicker__title a-button"}>
 					{this.renderNav()}
 				</Button>
 
 				<Button
+					type="button"
```
Comment (frankdsm) [inline packages/calendar/src/Datepicker/Nav.js]: htmlType="button"

**Pull request merged**